README.md

```diff
@@ -14,8 +14,7 @@ This website distinguishes between these fake mints and real ones, labeling them
 
 ![Screen Shot 2021-11-30 at 2 58 41 PM](https://user-images.githubusercontent.com/17352012/144141661-6c069a51-cd77-4a70-995c-1dde716656db.png)
 
-
-with this, in combination with filtering, the goal of the site is to be a tool to help discover and share alpha and hopefully prevent some people from being tricked into fomo buying something.
+With this, in combination with filtering, the goal of the site is to be a tool to help discover and share alpha and hopefully prevent some people from being tricked.
```

Comment:
I don't even know what "share alpha" means would be good if you provided more clarity

Reply:
totally